shub/config.py

```diff
@@ -16,6 +16,8 @@
                         pwd_hg_version, pwd_git_version, pwd_version)
 
 
+SH_IMAGES_PREFIX = 'images.scrapinghub.com/project/'
+SH_IMAGES_REGISTRY = SH_IMAGES_PREFIX + '{project}'
```

Comment:
I addressed it with https://github.com/scrapinghub/shub/pull/253/commits/e67d7f1cdfe645893746671701318d8753d33016, let me know wdyt.